include/ccf/tx_receipt_path.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/crypto/sha256_hash.h"
+
+#include <vector>
+
+namespace ccf
+{
+  struct TxReceiptPathStep
```

Comment:
Nice. I'd thought the `PathStep` needed to be separately visible, but I think that was because of an include order stemming from the should-have-been deleted copy noted below. I've now removed this file, and rolled this inside `TxReceipt`.